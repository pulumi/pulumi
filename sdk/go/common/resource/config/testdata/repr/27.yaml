my:outer:
  inner: 100
