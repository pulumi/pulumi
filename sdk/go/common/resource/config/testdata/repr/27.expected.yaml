decrypt:
  my:outer: '{"inner":100}'
paths:
  my:outer:
    value:
      inner: 100
    string: '{"inner":100}'
    redacted: '{"inner":100}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAW/4ITAAEFaW5uZXIFaW50NjQEAwD/yA==
    secure: false
    isObject: true
  my:outer.inner:
    value: "100"
    string: "100"
    redacted: "100"
    object: EBAABnN0cmluZwwFAAMxMDA=
    secure: false
    isObject: false
